draft-ietf-quic-transport.md

```diff
@@ -3688,11 +3694,13 @@ to adjust for any intentional delays, which is important for getting a better
 estimate of the path RTT when acknowledgments are delayed.  A packet might be
```

Comment:
I think we should start using 'introduced delays' instead of 'intentionally introduced delays' because now we have two cases where the delay isn't really intentional: alarms which wake up late and lack of availability of decryption keys.